package/homebrew/punktf-bin.rb

```diff
@@ -0,0 +1,19 @@
+class PunktfBin < Formula
+  version '1.0.0-alpha'
+  desc "A cross-platform multi-target dotfiles manager"
+  homepage "https://github.com/Shemnei/punktf"
+
+  if OS.mac?
+      url "https://github.com/Shemnei/punktf/releases/download/#{version}/punktf-#{version}-x86_64-apple-darwin.tar.gz"
```

Comment:
I don't think we will have a apple build at the start. This should probably be an error for now?

Reply:
Why not?

Reply:
We need to figure out cross-compiling and we also can't test the build afterwards

Reply:
dropped darwin support for now